examples/pna.py

```diff
@@ -19,8 +19,15 @@
 val_loader = DataLoader(val_dataset, batch_size=128)
 test_loader = DataLoader(test_dataset, batch_size=128)
 
-# Compute in-degree histogram over training data.
-deg = torch.zeros(5, dtype=torch.long)
+# Compute the maximum in-degree in the training data.
+max_degree = -1
+for data in train_dataset:
+    d = degree(data.edge_index[1], num_nodes=data.num_nodes, dtype=torch.long)
+    if d.max().item() > max_degree:
```

Comment:
This is a little bit cleaner :)
```suggestion
max_degree = max(max_degree, int(d.max()))
```